pkg/local_object_storage/internal/log/log.go

```diff
@@ -0,0 +1,19 @@
+package storagelog
+
+import (
+	"github.com/nspcc-dev/neofs-node/pkg/util/logger"
+	"go.uber.org/zap"
+)
+
+// a distinctive part of all messages
+const headMsg = "local object storage operation"
+
+// Write writes message about storage engine's operation to logger.
+//
+// Address should be type of *object.Address or string.
```

Comment:
how do logs of `*object.Address` look?

wouldn't it be better to use type switch?

Reply:
Type switch is done in `zap.Any`, `*object.Address` is processed as `fmt.Stringer`. I don't think we need to do explicit cast on our side.

Reply:
ok